src/main/resources/application.conf

```diff
@@ -22,6 +19,18 @@ mantis {
   # Uncomment this to specify, otherwise use the default implementation
   # secure-random-algo = "NativePRNG"
 
+  keyStore {
```

Comment:
Should we also make this change to the [`storage.conf`](https://github.com/input-output-hk/mantis/blob/security/minimal_password_length/src/universal/conf/storage.conf#L10) file in the universal config?